src/bucket.ts

```diff
@@ -765,4 +765,23 @@ export default class Bucket {
       aggregate: !!options.aggregate,
     });
   }
+
+  /**
+   * Retrieves bucket server endpoints.
+   *
+   * @return {Object}
+   */
+  get endpoint(): Record<string, Function> {
```

Comment:
Can you create an interface type and use that as the return type? So something like this:

```typescript
interface BucketEndpoint {
  history: () => string;
  group: (group: string) => string;
  // etc
}